core/src/main/java/org/commonjava/maven/ext/core/impl/ProjectVersioningManipulator.java

```diff
@@ -283,8 +284,8 @@ else if ( !changed && project.isInheritanceRoot() )
 
                         if ( newVersion != null && d.getVersion() != null )
```

Comment:
Do we need the check `d.getVersion() != null` here?

Reply:
It is passed into updateProperties so I'd suggest keeping it.